interp/notation.mli

```diff
@@ -320,3 +321,6 @@ val entry_has_ident : notation_entry_level -> bool
 (** Rem: printing rules for primitive token are canonical *)
 
 val with_notation_protection : ('a -> 'b) -> 'a -> 'b
+
+(** Conversion from bigint to int63 *)
+val int63_of_pos_bigint : Bigint.bigint -> Uint63.t
```

Comment:
Perhaps this belongs in the Uint63 module instead?  (Unless we want to keep it outside the kernel.)

Reply:
I think there are no `bigint`s in the kernel.